client/my-sites/stats/stats-card-upsell/stats-card-upsell-wpcom.tsx

```diff
@@ -32,9 +35,12 @@ const getUpsellCopy = ( statType: string ) => {
 		case STATS_FEATURE_DATE_CONTROL:
 			return translate( 'Compare different time periods to analyze your site’s growth.' );
 		case STAT_TYPE_VIDEO_PLAYS:
-			return translate( 'Discover your most popular videos and find out how they performed.' );
+			return getLocaleSlug()?.startsWith( 'en' ) ||
+				hasTranslation( 'Discover your most popular videos and find out how they performed.' )
+				? translate( 'Discover your most popular videos and find out how they performed.' )
```

Comment:
I thnk this will need to be cleaned up once teanslations are complete. Can you pls create a follow-up issue or something?

In the future, unless it's an urgent feature/request, then it's best to add the teanslation label, wait for translations to complete, and then merge.

Reply:
> it's best to add the teanslation label, wait for translations to complete, and then merge.

Thanks for clarifying the process! Will do.

Reply:
Translations are now complete, cleanup deployed: https://github.com/Automattic/wp-calypso/pull/93442